content/guides/tree-shaking.md

```diff
@@ -84,3 +84,5 @@ function(e,t,n){"use strict";function r(e){return e*e*e}t.a=r}
 /* ... */
 function(e,t,n){"use strict";Object.defineProperty(t,"__esModule",{value:!0});var r=n(0);console.log(n.i(r.a)(5))}
 ```
+
+T> Note, using the `--optimize-minimize` flag is NOT required for tree shaking to occur. Tree shaking will occur when using a production configuration. This can also be configured directly in a `webpack.config.js` file. Read [production build](/guides/production) for an in-depth guide on how to create a production config file.
```

Comment:
> However, I am unsure about the first part. Do you mean using something like babel-plugin-resolver when you say resolving using transpilers? I am not sure how transpilers work, but I am assuming loading your JS files with babel-loader does not resolve any imports, right?

Depends on how you have the `babel` and the `babel-loader` configured. With certain plugins, you'll need to set `{ modules: false }` in order to let webpack resolve the `import` and `export` syntax.

> What do you think about something along the lines of...

Definitely sounds better... I might reword/simplify as:

> Note that the `--optimize-minimize` flag enables tree shaking by including the `UglifyJsPlugin` behind the scenes. The `UglifyJsPlugin` can also be included manually in the `plugins` section of your configuration file. The plugin, combined with webpack's resolving of `import` and `export` statements, is what makes tree shaking possible. See the [production build guide](/guides/production) for more information.

What do you think?